### DISCUSSION TITLE
[General] <Title>Sir, I cannot join the discord server.

### DISCUSSION THREAD

**Discussion by ashik0121** [zkSync-Community-Hub/zksync-developers#231] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

hello i have some issue, i was in the discord channel and i don't know why i'm not in there anymore and i can't join it

### What is the specific issue or error you're encountering?

discord

### Can you share the error messages or logs you're receiving, if any?

discord

### Have you made any recent changes to the contract before encountering this issue?

discord

### Are there any external libraries or contracts that your contract interacts with?

discord

### Can you provide the relevant portions of your contract code where the issue is occurring?

discord

### Have you tried to isolate the problem, and if so, what were the results?

discord

### What steps have you already taken to try to resolve the issue?

discord

### Additional Details

discord

**Comment by AlbionaHoti** [comment, marked as answer]
Sir @ashik0121, please send an email to [support@zksync.io](mailto:support@zksync.io); this is not the right place to post these questions!
